x-pack/plugins/lens/public/datasources/common/field_item.test.tsx

```diff
@@ -5,42 +5,104 @@
  * 2.0.
  */
 
-import React, { ReactElement } from 'react';
-import { ReactWrapper } from 'enzyme';
+import React from 'react';
 import { act } from 'react-dom/test-utils';
 import { uiActionsPluginMock } from '@kbn/ui-actions-plugin/public/mocks';
 import { KibanaContextProvider } from '@kbn/kibana-react-plugin/public';
 import { dataPluginMock } from '@kbn/data-plugin/public/mocks';
 import { dataViewPluginMocks } from '@kbn/data-views-plugin/public/mocks';
 import { DataView, DataViewField } from '@kbn/data-views-plugin/common';
-import * as loadFieldStatsModule from '@kbn/unified-field-list/src/services/field_stats';
-import { FieldStats, FieldPopoverFooter } from '@kbn/unified-field-list';
-import { render, screen } from '@testing-library/react';
+import { loadFieldStats } from '@kbn/unified-field-list/src/services/field_stats';
+import { fireEvent, render, screen } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
-import { EuiLoadingSpinner, EuiPopover } from '@elastic/eui';
 import { InnerFieldItem, FieldItemIndexPatternFieldProps } from './field_item';
 import { coreMock } from '@kbn/core/public/mocks';
-import { mountWithIntl } from '@kbn/test-jest-helpers';
 import { fieldFormatsServiceMock } from '@kbn/field-formats-plugin/public/mocks';
 import { IndexPattern } from '../../types';
 import { chartPluginMock } from '@kbn/charts-plugin/public/mocks';
 import { documentField } from '../form_based/document_field';
+import { KibanaRenderContextProvider } from '@kbn/react-kibana-context-render';
 
 jest.mock('@kbn/unified-field-list/src/services/field_stats', () => ({
   loadFieldStats: jest.fn().mockResolvedValue({}),
 }));
 
```

Comment:
Another idea for small cleanup, I think you could pass a reference to a `loadFieldStatsMock` like this...

```suggestion
const loadFieldStatsMock = jest.fn().mockResolvedValue({}); 
jest.mock('@kbn/unified-field-list/src/services/field_stats', () => ({
  loadFieldStats: loadFieldStatsMock,
}));

```

Then whenever we access the mock rather than type casting it to `(loadFieldStats as jest.Mock).someMethod` we could just use it as is `loadFieldStatsMock.someMethod`. Also no need to import `loadFieldStats`.

I'm 98% sure this would work. But just an idea take it or leave it 😄.

Reply:
This unfortunately doesn't work as it throws 'Cannot access before initialization error'. probably there's a way to fix it, but I couldn't find it quickly :(